example/docker-compose.yml

```diff
@@ -34,6 +34,8 @@ services:
       - env.d/learninglocker
     expose:
       - "8080"
+    volumes:
+      - "./storage:/storage"
```

Comment:
Can we use defined environment variable to avoid repetition? And use `/storage` as the default:

```yaml
volumes:
  - "./storage:/${FS_LOCAL_ENDPOINT:-/storage}"
```

Reference: https://docs.docker.com/compose/environment-variables/

Reply:
:thinking: maybe this variable won't be defined as loaded in an env file when docker-compose will run.

Reply:
> thinking maybe this variable won't be defined as loaded in an env file when docker-compose will run.

Indeed, this variable is defined in the container, not in the host machine.